publish-sdk

```diff
@@ -10,13 +10,21 @@ for opt in "$@"; do
       --registry=*) REGISTRY="${optarg}" ;;
       --sdk-name=*) SDK_NAME="${optarg}" ;;
       --short-sha=*) SHORT_SHA="${optarg,,}" ;;
+      --skip-manifest=*) SKIP_MANIFEST="${optarg,,}" ;;
+      --only-manifest=*) ONLY_MANIFEST="${optarg,,}" ;;
```

Comment:
Can we error out if `SKIP_MANIFEST` and `ONLY_MANIFEST` are both set, or exit immediately with "nothing to do" in that case?